### DISCUSSION TITLE
use of offline

### DISCUSSION THREAD

**Discussion by wenushkamallikarachchi** [google/model-viewer#2145] [General]
how to download the model-viewer file to use offline

**Comment by felipe-chamas** [comment]
You can download the files from [unpkg](https://unpkg.com/@google/model-viewer/dist/)

**Reply by felipe-chamas** [reply]
Ex: https://unpkg.com/@google/model-viewer/dist/model-viewer.js
Hit `Ctrl + S`

**Reply by wenushkamallikarachchi** [reply]
![s](https://user-images.githubusercontent.com/47137116/112018765-df9c2c80-8b54-11eb-8a46-5d06ce98d722.JPG)

Now there is a Cors error
Please help me

**Reply by felipe-chamas** [reply]
If you are using simple HTML to import, you can put the file within the folder of the HTML and reference it relatively:
```html
<script type="module" src="./model-viewer.js"></script>
```

**Reply by felipe-chamas** [reply]
If it's in a different folder, you can serve the folder with `npx live-server` and reference as:
```html
<script type="module" src="http://localhost:8080/model-viewer.js"></script>
```
I was never able to make it work with `file://`

**Reply by wenushkamallikarachchi** [reply]
its work for me
